Updated es_es.json

FrannDzs opened the pull request:
@muriplz ~~You can check from this template which is the latest version, there are quite a few fixes still to be done :,(~~
I have also added some lines that have not yet been completed by the developer but I added them to not fall too far behind, if explanations are missing or are in English it is because of the new stuff that is being redone.

In spite of this, it can be merged at the moment.

paladini-maker committed:
Updated es_es.json

paladini-maker committed:
Updated es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Revised and improved translation of the ponder section

paladini-maker committed:
Updated ponder section

More fixes.

paladini-maker committed:
Updated ponder section

More fixes...

paladini-maker committed:
Updated es_es.json

Many arrangements
General improvements to the ponder section

After a couple of hours of work it looks like this is up to date. =)
@muriplz You can check from here and test the template, I finished checking it in game and it looks good.

muriplz commented:
Hello, how do i merge or make a pullrequest of your pull request?

SpottyTheTurtle commented:
Only contributers can merge

paladini-maker committed:
Update es_es,json

Ready for review

FrannDzs commented:
> Hello, how do i merge or make a pullrequest of your pull request?

Hello, you must delete the original fork you have from Create and remake a fork from my branch....

muriplz commented:
I did it, but then how i pull request to your branch? takes me to the main branch "Creators-of-create"

FrannDzs commented:
You should look for my branch there

![image](https://user-images.githubusercontent.com/76663897/122699717-44ccf000-d220-11eb-8724-0ba97b7d15bc.png)

FrannDzs commented:
@muriplz You have done it before 😄

paladini-maker committed:
Update es_es.json

Fixed some items such as cinder flour, now called Nether flour, because it is made with Nether rock and is used to create blaze cakes.

paladini-maker committed:
Merge branch 'Creators-of-Create:mc1.16/dev' into mc1.16/dev
